code/modules/mob/living/carbon/xenomorph/strains/castes/drone/healer.dm

```diff
@@ -124,9 +128,9 @@
 	adjustBruteLoss(amount * damage_taken_mod)
 	use_plasma(amount * 2)
 	updatehealth()
-	new /datum/effects/heal_over_time(target_xeno, amount, 10, 1)
-	target_xeno.xeno_jitter(1 SECONDS)
-	target_xeno.flick_heal_overlay(10 SECONDS, "#00be6f")
+	new /datum/effects/heal_over_time(target_xeno, heal_amount = amount)
+	target_xeno.xeno_jitter(5 SECONDS)
```

Comment:
You clearly wanted to make jitter last 1 second, and flick 5, not the other way around

Reply:
chat is real